src/box/sql.c

```diff
@@ -283,7 +283,7 @@ sql_space_info_new(uint32_t field_count, uint32_t part_count)
 	info->parts = part_count == 0 ? NULL :
 		      (uint32_t *)((char *)info->coll_ids + colls_size);
 	info->sort_orders = part_count == 0 ? NULL :
-			    (uint32_t *)((char *)info->parts + parts_size);
+			    (int32_t *)((char *)info->parts + parts_size);
```

Comment:
Nit: Should be `(enum sort_order *)`?

Reply:
The change has been dropped from the patchset.